tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
+	if (check_screen('encrypted_volume_activation_prompt')) {
+	    record_soft_failure 'bsc#989770';
+	    send_key 'alt-c';
+	}
```

Comment:
you can shorten the wait time by calling `assert_screen` with multiple tags, i.e. in this case the next tag that should be expected afterwards in "good" case as well as the 'encrypted_volume_activation_prompt' and use `match_has_tag` to check if the prompt reappears but this will work for now.